lexicons/com/atproto/server/checkAccountStatus.json

```diff
@@ -0,0 +1,38 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.server.checkAccountStatus",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Returns the status of an account, especially as pertaining to import or recovery. Can be called many times over the course of an account migration.",
```

Comment:
I assume this is auth-required and only the account can call on themself; should probably add that to description.

should we allow admins to call for any account? might help with debugging. in that case would want an optional accountDid param, or something like that. could come back to this later.

Reply:
yup added to description 👍 

i think we if we open this up for admins, then we can add the accountDid param 👍